### DISCUSSION TITLE
When I restart clickhouse, a large number of files are detached,

### DISCUSSION THREAD

**Discussion by hert525** [ClickHouse/ClickHouse#47894] [General]
When I restart clickhouse, a large number of files are detached, all of which are tables of replacingmergetree structure. How can I avoid this problem?

**Comment by den-crane** [comment]
Do you mean after a restart you observe files in the `detached` folder?
What are the prefixes of the files? `ignored_` / `cloned_` ?

**Reply by hert525** [reply]
![image](https://user-images.githubusercontent.com/19421393/227123334-6b435bf2-5c1d-45b2-9d1b-b75a36bd6273.png)
服务器没有宕机过，clickhouse重启后很多文件校验失败，然后成为了detached

**Reply by hert525** [reply]
The server has not been down before. After the clickhouse restarts, many files fail to be verified and become detached

**Reply by den-crane** [reply]
Did you restart a hardware? Check `uptime` (of linux)

**Reply by hert525** [reply]
No, I never restarted hardware
